File: app/src/main/java/com/github/RooneyAndShadows/lightbulb/pickersdemo/utils/color/AppColorUtils.java
package com.github.rooneyandshadows.lightbulb.pickersdemo.utils.color;


import com.github.rooneyandshadows.lightbulb.dialogs.picker_dialog_color.ColorPickerAdapter;
import com.github.rooneyandshadows.lightbulb.pickersdemo.utils.color.colors.DemoColors;

import java.util.ArrayList;

public class AppColorUtils {

    public static int getColor(IDemoColor desiredColor) {
        return desiredColor.getColor();
    }

    public static ArrayList<ColorPickerAdapter.ColorModel> getAllForPicker() {
        ArrayList<ColorPickerAdapter.ColorModel> result = new ArrayList<>();
        for (DemoColors color : DemoColors.values())
            result.add(new ColorPickerAdapter.ColorModel(color.getColorHex(), color.getName()));
        return result;
    }
}

File: app/src/main/java/com/github/rooneyandshadows/lightbulb/pickersdemo/utils/color/AppColorUtils.kt
package com.github.rooneyandshadows.lightbulb.pickersdemo.utils.color

import com.github.rooneyandshadows.lightbulb.dialogs.picker_dialog_color.ColorPickerAdapter.ColorModel
import com.github.rooneyandshadows.lightbulb.pickersdemo.utils.color.colors.DemoColors
import java.util.ArrayList

object AppColorUtils {
    fun getColor(desiredColor: IDemoColor): Int {
        return desiredColor.color
    }

    val allForPicker: ArrayList<ColorModel>
        get() {
            val result: ArrayList<ColorModel> = ArrayList<ColorModel>()
            for (color in DemoColors.values()) result.add(ColorModel(color.colorHex, color.colorName))
            return result
        }
}